src/createReferenceObject.js

```diff
@@ -1,19 +1,25 @@
-const createBoundingBox = require('./createBoundingBox.js');
+const createBoundingBox = require('./createBoundingBox');
 
 //Create a reference object for an element
-module.exports.getRef= function (cyElement, userOptions) {
+module.exports.getRef = function (target, userOptions) {
```

Comment:
This function should not have if-else code for nodes/edges/core.  It should have everything passed into it.  Only the topmost functions should have core/node/edge-specific code.

Reply:
Removed